### ISSUE TITLE
[FEATURE]: a feature to support chat from You.com

### ISSUE THREAD

**Issue by yu2chuan** [drl990114/MarkFlowy#123]
**Is your feature request related to a problem? Please describe.**
I personally think the chat from You.com isn't bad. It could be another option in the linebyline.

**Describe the solution you'd like**
Not only support ChatGPT but also chat from You.com

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by halodong** [comment]
@yu2chuan Do you mean `You` here?

<img width="306" alt="image" src="https://github.com/linebyline-group/linebyline/assets/48054715/8553f3aa-0e49-4619-8172-2611b441ed5a">

**Comment by yu2chuan** [comment]
I mean a chat provided by You.com. They call it YouBot.
https://you.com/search?q=who+are+you&tbm=youchat&cfr=chat

**Comment by halodong** [comment]
Good idea, this feature may indeed be useful for some users, and later it will support some settings to give users more choices.

**Comment by drl990114** [comment]
Currently, accessing external services is supported via configuring the `baseURL` and `request headers`. This issue is temporarily closed. If you have other requirements, please submit a new one.
